src/modules/IscsiClientLib.rb

```diff
@@ -200,37 +200,52 @@ def getFirmwareInfo
       bios_info["stdout"] || ""
     end
 
+    # Takes the output of 'iscsiadm' either in 'node', 'session' or 'fw' mode, i.e.
+    # output where info is seperated by '=', like
+    # iface.transport_name = tcp
+    # iface.hwaddress = 42:52:54:00:8b:6a
+    # iface.bootproto = DHCP
+    #
+    #  @param [String]   stdout (where info is seperated by '=') of iscsiadm command
+    #  @return [Hash]    command output converted to hash
+    #
+    def nodeInfoToMap (stdout)
+      retval = {}
+      return retval if stdout.empty?
+
+      stdout.split("\n").each do |row|
```

Comment:
you can use `lines` instead of `split("\n")`. Only difference is that lines keep trailing "\n", but strip below remove it.
```
irb(main):004:0> "a\nb".split("\n")
=> ["a", "b"]
irb(main):005:0> "a\nb".lines
=> ["a\n", "b"]
```